src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
+In this exemple, the [Description] tag is set as step test case in the allure-reporter tool.  
```

Comment:
I suggest 'The [Description] tag is the step test case.

Reply:
changed

Reply:
changed